app/models/post.rb

```diff
@@ -1,5 +1,6 @@
 class Post < ApplicationRecord
   belongs_to :subject
+  has_many :comments, as: :commentable
```

Comment:
Whats with the `commentable`

Reply:
It is required for the polymorphic association to work.